coral-service/settings.gradle

```diff
@@ -0,0 +1,9 @@
+pluginManagement {
+  repositories {
+    maven { url 'https://repo.spring.io/milestone' }
+    maven { url 'https://repo.spring.io/snapshot' }
+    gradlePluginPortal()
+  }
+}
```

Comment:
So the `settings.gradle` file is to define all included submodules, so you can only have one `settings.gradle` file per multi-module project (i.e only one in the entire Coral repo). I've discussed this with Lei but we had issues configuring the `coral-service` module as a submodule of `coral`, which would be ideal. Ultimately, this means in the meantime we need this `settings.gradle` file in `coral-service` because it's being treated as it's own multi-module project.

Integrating `coral-service` as a submodule of `coral` should be considered in future work.

Reply:
@wmoustafa I also think it's reasonable to treat `coral-service` as a separate module instead of a coral submodule, given `coral-service` is not like other coral modules which do translation.